gdk4-wayland/sys/Cargo.toml

```diff
@@ -13,6 +13,9 @@ version = "0.2.0"
 [package.metadata.system-deps.gtk4_wayland]
 name = "gtk4-wayland"
 version = "4.0.0"
+
+[package.metadata.system-deps.gtk4_wayland.v4_4]
+version = "4.4"
```

Comment:
here's where you can change the required version number

Reply:
That's what I thought, but thought it was generated. it's magic it isn't, or not fully :)